xml/System.Security/SecurityTransparentAttribute.xml

```diff
@@ -40,6 +40,8 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+ Partially trusted code is no longer supported. This attribute has no effect in .NET Core.
```

Comment:
```suggestion
> [!IMPORTANT]
> Partially trusted code is no longer supported. This attribute has no effect in .NET Core.
```
Rather than simply adding it as the first paragraph of the Remarks section, I think it should be in a note box for greater prominence. I thought an IMPORTANT box is appropriate, though it could also be a NOTE box.